### DISCUSSION TITLE
A bigger picture rework to allow for more diversification in Human Space

### DISCUSSION THREAD

**Discussion by xX-Dillinger-Xx** [endless-sky/endless-sky#9914] [Ideas]
I started this in 'Discussions' for ease of ...well, discussion. I hope one day it will get moved to a PR, but PR's aren't the place for discussing an idea. If anyone want's to start a PR, at anytime, feel free. I probably won't feel comfortable enough to do one myself for quite awhile.

There seems to be a growing consensus that there are some issues around diversification in human space. 
I would like to throw my hat into the ring with a series of ideas for improvements. It's a bigger picture kind of idea.
I'm taking everything I've read and the knowledge that more human space campaigns are coming, into account. Many of the ideas are just me thinking out loud. It's more about getting a conversation started and get people thinking, in an organized fashion. Good discussion fuels inspiration and innovation.

All I ask, is to keep an open mind, see where this leads, and don't get bogged down in minor details at first. I'm a smart guy who just wants to help but lacks the skills to do it directly. I'm aware that some of this will require some re-balance work but, if done right, will set the stage for some good future stories and missions, etc. 
Additionally, it may cause some issues with save files and plugins, but to that I would say that at some point, that may become a necessary evil in order to move forward anyway. I don't think plugin developers would mind too much if it's for the betterment of ES, as a whole. As for save files, I certainly wouldn't mind starting from scratch for a whole new experience. Additionally, I don't think you should let the fear of breaking a plugin or breaking backward compatibility hinder future development. 

We've all seen multiple suggestions:
- adjust stats balance
- don't adjust stats balance
- hide certain outfits for a later reveal
- add even more ships and outfits
- bring in outfits from other species
- etc.etc.
I would like to see if it's possible to bring some of these ideas together to create one big cohesive idea, as a group. Right now it looks like a bunch of people working on separate solutions, on there own. Individually, some don't make sense, but when they become part of a bigger picture, they work.

My idea revolves around the redistribution of existing assets, and maybe adding a few more. I'm referring to ships and outfits.

Currently, there is a huge imbalance (I don't mean stats) of ships and outfits amongst the 3 main factions. The Republic has the Navy. Free Worlds gets 2 new ships and 3 new weapons. The Syndicate doesn't get much of anything except regenerators and nukes, which aren't even identified as theirs. Also, what they do have isn't restricted in anyway and what they get has a very short in game life span. If the Syndicate is supposed to keep secrets where are all the results of their secret work and projects? Being constantly raided buy the Korath, I'm sure they have retrieved some good stuff and as a result have some more interesting tech available at some point.

Now pirates are stealing things all the time and it makes sense they might have some good engineers. Now, are these engineers payed, are they prisons, or both?   I find it very believable for them to have some modified ships (i.e. Marauders) and outfits, even some tech from other species. They become our hints to what might be out there. Maybe, they even have stolen prototypes that they have modified to suit their needs. < more hints. Also, _prototypes_ would explain the weaker versions needed to maintain game balance. While hinting that maybe the player will get access to better versions later.

Part one of my idea would be to try to reassign and redistribute some of the 80 ships in human space. The idea is to try and balance things out amongst the three factions. Pirates, could benefit as well. Discuss this topic in the next post down.

Part two would involve all the outfits. There are certain outfits that could be made faction specific and even locked behind licenses.   We can discuss this topic two posts down.

Comments about the idea in general can go in the 3rd post down. I didn't know you wouldn't be able comment directly under the 1st one

**Comment by xX-Dillinger-Xx** [comment]
### SHIPS
There are, if my count is correct, 80 ships in human space, if you count Marauders. Some of these are pirate versions and aren't currently in game. 
- 9 Marauders non purchasable but capturable without any additional repercussions and very public
- 9 Republic Navy specific that require some sort of Navy license and are very public
- 2 Militia (free worlds) Requires militia license
- the Bactrian (locked behind a Deep mission)
- the Kestrel (locked until later in game)
- 0 Syndicate locked ships
- 58 many are free to purchase anywhere by anyone. This 58 also includes the currently unavailable pirate versions, which I don't know the plans for. 

What could we do to redistribute some of the 58? 
- The Protector and Vanguard are Syndicate produced. 
  - maybe give one of these a little love and lock it behind a license. 
- make a Syndicate Securities variant of the Quicksilver to rival the Rep. Gunboat. 
  - lock it behind a license.
  - I've never even bought a Quicksilver, only ever captured pirate owned ones. 
  - Another odd thing is pirates have a marauder versions but Syndicate Security only has the civilian version
- maybe take a couple of the currently unused 'pirate versions' and give them to the FW or Synd.

Just like anything Navy, license locked Syndicate and/or FW ships are capturable, but that comes at a cost unless becoming a pirate is your goal.

**Reply by lumbar527** [reply]
Just throwing out some of my ideas in response.

> * The Protector and Vanguard are Syndicate produced, maybe give one of these a little love and lock it behind a licence.

If the Protector and Vanguard are locked then perhaps other heavy warships could be also. The falcon is actually produced by Tarazed so that could also make sense. I think the Leviathan might not need a license because it's so old.

> * make a Syndicate Securities variant of the Quicksilver to rival the Rep. Gunboat. Lock it behind a licence. (I've never even bought a quicksilver, only ever captured pirate owned ones.)

Good idea!

**Reply by Zitchas** [reply]
I'm all in favour of heavy warships being hard to get. Mediums too, for that matter. I don't even think they need buffing. They should all need licenses for regular purchase,  and very hard to capture from anything less than them.

There only things being freely available on the open market should be civilian ships. Honestly, the Clipper and other "freighters with good combat potential", along with a variety of light warships should be the best combat suis that can be purchased without earning some trust from their respective shipyards and governments.

**Reply by xX-Dillinger-Xx** [reply]
I see Quantumshark is working on [buffing the Quicksilver](https://github.com/endless-sky/endless-sky/pull/9850), which works nicely with what I have suggested to do with the Quicksilver for Syndicate Security
We could always swap to Proton guns to stick to the Synd, theme.

**Reply by bene-dictator** [reply]
I have thought in the past of making Particle Cannons restricted to the Navy / Syndicate but leaving Proton Guns a civilian outfit.

**Reply by xX-Dillinger-Xx** [reply]
I was looking at the current Marauder Quicksilver. Could we copy that and assign it to Syndicate Security? 
It would need to be tweaked it a little. I like the idea of the reverse thrusters but that doesn't fit with heavy lasers so it would need different guns.

**Reply by Hurleveur** [reply]
> I have thought in the past of making Particle Cannons restricted to the Navy / Syndicate but leaving Proton Guns a civilian outfit.

I almost prefer the proton to it anyway honestly, especially in turret form its almost busted

**Comment by xX-Dillinger-Xx** [comment]
### OUTFITS 
There are a lot of outfits we can play around with. 
Let me use the Synd. regenerators as an example. 
- Identify it as Syndicate (currently it is not)
- make the small one available from the start
  -  it would need to match the equivalent "D" version. Not exactly, but close enough to be similar.
- maybe add a 3rd version
- other larger versions should be seen in Syndicate outfitters but not available without a license.
  - this exposes the player to possibility they may be able to get a 'Syndicate Securities License' some day. Which also leads player to question; _*"How do I get that and I wonder what else I might get with it?"*_ This is an incentive to consider joining them when the opportunity does present itself.

When I was chatting with bene-dictator, and he had this question, _*"Why wouldn't the Rep. be using Synd regens?"*_ Well, because there wouldn't be a big enough advantage. If the only one publicly available is just the small one, they wouldn't want to pay for them when they could just use their own. Now the player might see an advantage to purchase a small one. The Syndicate won't sell the Republic the larger ones, for whatever reason. Maybe they want to maintain some sort of advantage, just in case. 

Hecter94 suggested this; _*"Armageddon Core could be buffed above the Fusion Reactor and then made Navy-exclusive."*_ in a different topic.

I just covered one example but please suggest any that you feel could be altered or reassigned for any faction. I think you get the whole idea.

**Reply by xX-Dillinger-Xx** [reply]
> **bene-dictator:** I have thought in the past of making Particle Cannons restricted to the Navy / Syndicate but leaving Proton Guns a civilian outfit.

Would P. Cannons be restricted to both or only one?

**Reply by bene-dictator** [reply]
The Navy make more use of particle cannons, it'd make more sense for the Syndicate to use proton cannons generally, but their elite corps might use particle cannons.

**Reply by Hurleveur** [reply]
Gameplay wise if we lock the particle cannon I think we should lock the proton gun too, as I actually think it's comparable and almost better

**Reply by bene-dictator** [reply]
Something tells me that a souped-up version of the proton gun (which replaces the existing one) might be useful, if we considered that route. _Then_ we have a civilian-grade proton gun.

**Reply by Hurleveur** [reply]
I think it'd make sense to have the syndicate licenses be buyable, actually, though at a fair price ofc

**Reply by bene-dictator** [reply]
Random idea: Korath-inspired plasma engines might also be interesting. Might be an upgrade to standard Human plasma engines. Navy ships use ion engines only then move to atomics, so perhaps these plasma engines could be the the Syndicate's hallmark. Or some other sort of engine (e.g. Zoura's Outfits Expanded electrostatic engines, which are derivatives of ion engines). I do realise that Delta V are the main producers of plasma engines and the Syndicate generally make ion engines, but ships associated with the Syndicate seem to use plasma engines (Star Barge, Quicksilver) so why not?

Though, three engine types for one species is already quite a lot, so another one would be quite a bit on top...

**Reply by Hurleveur** [reply]
Those new engines would also probably make the hai ones obselete

**Reply by xX-Dillinger-Xx** [reply]
>bene-dictator: Random idea: Korath-inspired plasma engines might also be interesting.

I also like the idea of Korath- inspired outfits. However, seeing as the only ships the Synd. see are the ones that come to human space, I think we should stick to **_only_** items that come on from `olofez . I don't think  it would make sense for the Synd. to able to disable a Palavret or a Rano'erek. Or do they now? Maybe later in game?

What about a modified or reverse engineered 'fire-lance' ? Longer range but weaker and maybe slightly unreliable. I have a longer reasoning if anyone is interested. 

>Hurleveur: I think it'd make sense to have the syndicate licenses be buyable, actually, though at a fair price ofc

Why buyable, we don't have any other buyable licenses do we? Maybe, buyable after a certain point in the story could make sense. The Synd. are all about money, correct?

**Reply by lumbar527** [reply]
The Syndicate have disabled a few larger Korath vessels because they get some jump drives. They probably didn't get many though.

**Reply by xX-Dillinger-Xx** [reply]
Good point. I never thought about where they came from. Ok maybe only Korath outfits from Palavrets, Rano'erek and `olofez. 
So, based on the updated knowledge, my question is why isn't there Korath inspired tech in Syndicate space? Why only jump drives?

How would we stop the Korath inspired tech from spreading across human space? Maybe, they aren't for sale, and are really hard to plunder. The Syndicate added security measures that render them useless if plundered.

**Reply by bene-dictator** [reply]
> Why buyable, we don't have any other buyable licenses do we? Maybe, buyable after a certain point in the story could make sense. The Synd. are all about money, correct?

There _used_ to be licenses for large ships that the player had to purchase, but MZ removed them because he felt they were not a meaningful blocker. However, I don't see a problem here, and besides, it kind of serves a narrative purpose that the Syndicate would charge you before you can buy their goods.

> So, based on the updated knowledge, my question is why isn't there Korath inspired tech in Syndicate space? Why only jump drives?
How would we stop the Korath inspired tech from spreading across human space? Maybe, they aren't for sale, and are really hard to plunder. The Syndicate added security measures that render them useless if plundered.

Well, as the Free Worlds story describes, the Syndicate did a decent job of hiding the Korath from the public. Korath tech is a bit higher level than Hai tech so it's not inconceivable that there were little results from their study, I still don't see why there couldn't be any; it'd probably be a large plot point in the Syndicate campaign.

**Reply by xX-Dillinger-Xx** [reply]
I think some Korath inspired _prototypes_ might be an additional way to go. Being _prototypes_ , they wouldn't be as good as the real thing (helps maintain balance) and would help to slightly discourage farming, in favour of seeking out the real thing. 

Why prototypes? Why wouldn't the Synd just use the captured tech directly?
> the Syndicate did a decent job of hiding the Korath from the public.  

So, I guess if they are hiding the Korath's existence then they really wouldn't want to risk exposure by using the tech directly. They would want to reverse engineer things to make it "look like" it was all their own work.  It fits the **_Syndicate keep secrets_** theme.

**Reply by Zitchas** [reply]
> There _used_ to be licenses for large ships that the player had to purchase, but MZ removed them because he felt they were not a meaningful blocker. However, I don't see a problem here, and besides, it kind of serves a narrative purpose that the Syndicate would charge you before you can buy their goods.

That was before my time, but I also disagree with it. While purchasable licenses wasn't much of a blocker (money is easy to get) it also wasn't a problem, either. What it was, at least to me, was a goal, an achievement, and gave a sense of progress and moving up in the world. It also reinforced the whole FW complaint about Republic bureaucracy, too. It can also be a bit of a "guide" for new players. Right out the gate as a brand new player, you probably shouldn't be buying heavy rockets, for instance. Once you've been playing long enough to purchase the "heavy ordinance license," though, you are probably readier to expand your options.

So in my view it was a win for:
- providing some subtle guidance,
- providing in-game lore,
- generating empathy with the FW,
- giving the player some goals that aren't campaign or Get Bigger Ship related
- give a sense of achievement
- can act as a kind of "tax" on whatever is locked behind those licenses.
- opportunity to make pirate outfitters/shipyards more appealing because they obviously won't check for licenses. (It'd be great if we could make an outfit count as illegal so long as the appropriate license wasn't present. Great for pirate gameplay)

**Reply by xX-Dillinger-Xx** [reply]
Sounds like we have 4 votes for adding buyable license's. They shouldn't be too cheap.
If I tie in my other idea of having "combat rating" mean more in the game, you could also add a lock that prevents buying certain ones unless you have a high enough rating. You could take it even one step further and  also look at reputations. So you would need money, skills, and a good reputation to be able to buy certain licenses. 

Another idea; a Mining License that allows you to legally sell high value mined goodies. The low value stuff doesn't need a license. The Mining License could also come with a list of systems that are known to have high value asteroids. And, again pirate systems don't care because _"they obviously won't check for licenses."_

**Reply by Zitchas** [reply]
I think my preference would be to keep combat rating separate from licenses, it perhaps be an alternate way to get them.

For example, have some generic Navy anti-pirate patrols that get bigger and tougher as your combat rating goes up, and at various points the Navy contact issues you the various licenses as rewards. So it would be a different way to get them for those who are good with combat and not so good with cash flow.

**Reply by esksp** [reply]
> There _used_ to be licenses for large ships that the player had to purchase, but MZ removed them because he felt they were not a meaningful blocker.

What is the difference between paying for a license + the ship, and just paying more for the ship?
I would suggest to **increase all the ship prices**, a lot. I don't know what the in-game currency's buying power is, but **in our world you wouldn't buy a (sea) container ship hull for only 240k (Hauler II). It would rather cost 2.4 million.**
Multiplying actual prices by 10 would make them much more realistic, and would keep the player longer on smaller ships, without the need for artificial invisible barriers like licenses. Licenses make only sense where story-wise it would indeed make sense to have licenses (like military grade equipment) anyway. Don't spam the player with licenses, some people might collect them but for others it's clutter in the outfitter window (why are they there anyway? They belong in the player information panel).

Another thing to possibly implement would be **preferential pricing**; Meaning **if you are popular with some faction, you pay less** (up to -20%). On the contrary, if you're not much appreciated, you pay through the nose (up to +50%).

Just random ideas.

**Reply by ziproot** [reply]
The problem with increasing ship prices is that it buffs capturing, one of the most powerful ways to make money, and nerfs everything else. I actually think prices should be *decreased* for this reason.

EDIT: Licenses could also help with the capture ladder as they would make it so that you are discouraged from capturing heavier ships until you have purchased the license for it. On the other hand, they may make this worse, as alien ships would likely not be license gated in the same way. I am specifically thinking about korath automata here, as there are already people who have jumped straight to korath automata, and this would be encouraged even more if it meant you didn’t have to spend time acquiring licenses.

**Reply by danaris** [reply]
There _are_ other ways to mitigate the buff to capturing: the simplest is to push the depreciation minimum percent further down.

**Reply by esksp** [reply]
> The problem with increasing ship prices is that it buffs capturing

But there is already a powerful mechanism to prevent this: **Depreciation**... (as danaris already said above)
Just lower the amount of money one can get for used ships, **doubly so if they are captured** (damages, missing papers, etc...). Or **make so that only shady places (pirate systems) buy them** (after all they're stolen).

Money is relatively easy to make one way or another (mining is surprisingly efficient too). The solution is _not_ to make everything even cheaper (i.e. requiring less money), on the contrary, everything should get _more expensive_.
Capturing ships is very lucrative right now, but if they become hard to sell and yield less money the player will turn to something else.

> until you have purchased the license for it.

I strongly dislike licenses as go-to solution to game balance.: It's clunky, lazy, and feels wrong, like invisible barriers in FPS.
Licenses should only be used where they make some real world sense, like for instance for buying military equipment.

**Comment by xX-Dillinger-Xx** [comment]
### Main Comment Discussion: 
I missed the fact you can't directly comment under the first post.

**Reply by xX-Dillinger-Xx** [reply]
I see some progress is being made in adding some new planets etc. in PR https://github.com/endless-sky/endless-sky/discussions/9817 and PR https://github.com/endless-sky/endless-sky/pull/9917. This idea, in conjunction with these new additions, might inspire some new plans for human space. Some are planned for use by pirates but I do see [Taylor](https://private-user-images.githubusercontent.com/115441627/312649216-280e8b68-5645-47d6-bbf2-0d8ba5baee75.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA5NzE5MjEsIm5iZiI6MTcxMDk3MTYyMSwicGF0aCI6Ii8xMTU0NDE2MjcvMzEyNjQ5MjE2LTI4MGU4YjY4LTU2NDUtNDdkNi1iYmYyLTBkOGJhNWJhZWU3NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMyMFQyMTUzNDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xNjMyYTUxMmVhZTNjNDA5OWM5NjE0YjFiMWU0OGQ4YjUxNDg0NDUwYmE0OGE1NmFiMmI1Zjg4NGU4OTUyMjExJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.xvcjELbuuH4uGW_J7sRQq0xxFKdAHVChLytE_EOsN5Q), a moon in the Sheratan system, is going to have a large restricted Syndicate Military base.

**Reply by xX-Dillinger-Xx** [reply]
I was re-reading some older discussions and found a rather lengthy one in my other 'discussion' about battles being too short.
It starts with a comment by Quantumshark [here](https://github.com/endless-sky/endless-sky/discussions/9862#discussioncomment-8727665).

The short version (TL:DR) We need _more_ diversification to allow the player to tweak their ships to suit their preferences. This can allow them to gain an advantage and introduce them to the concept of mixing and matching faction-specific items for later in the game. 
Currently, we do not have this. 

Quantumshark and bene-dictator are both suggesting we need more diversity, of outfits and ships, in human space. I would suggest you go and read their comments for clarity. I don't see any point in duplicating them here.

Hecter94 is suggesting that adding some, might be good, but not to add too many. This is his key point; _"If we were going to introduce unique faction-specific equipment and load-outs, I'd limit it to just a few per faction, enough to give the player a slight edge and introduce them to the concept of faction-specific outfits without it being an overwhelming amount of information to process right at the start of the game."_  I tend to agree with this point.

bene-dictator also had this to say; 
_Endless Sky's development goal has been, for a while now, the three main human campaigns, and after all this time we still only have one of at least three implemented. We will need some more faction specific equipment for at least one of the two other campaigns – though Navy ships might be good enough for the Republic campaign, there is nothing much we can't normally access ingame through FW that the Syndicate can't give you (namely nukes, ionic afterburners, and regenerators)_
This helps support my whole idea behind this 'Discussion'. Now would be a good time to consider my ideas, while the other 2 campaigns are still being worked on.

**Comment by VaelophisNyx** [comment]
personally I have two thoughts:
1. Human space has way, way more ships than anywhere else
2. there's not much of a good progression of Light ships compared to medium or heavy. You're pushed along to bigger and better pretty quick because, without those ships you can't carry enough cargo, or gun, or shield, or whatever. Wolf-pack tactics of 30 small ships also rarely work because of just how fragile they are and how incompetent the fleet AI can be with ships that aren't super durable

2 definitely is fed into by the ever-present "big ship better" issue, which while they should be more combat capable, even post movement rework, the combat capability of a ship scales non-linearly with its hull, internal volume, and mounts (esp. for human ships)
